### ISSUE TITLE
[Bug] Crashes should be contained to a single tab

### ISSUE THREAD

**Issue by pwd-github** [mozilla-mobile/fenix#2863]
## Steps to reproduce
1. Open ten GitHub tabs (various repositories, issues, etc) 
1. Attempt to file a new issue 
1. Attempt to attach a screenshot
1. Attempt again because it will fail the first time
1. Press back

### Expected behavior
You return to the form you was filing an issue on. 

### Actual behavior
You return to an about:blank tab and find that the other GitHub tabs are now about:blank tabs too. 

### Device information
* Android device: Android 8, LG G6
* Fenix version: 1.0.1921

**Comment by PopeRigby** [comment]
Having a similar issue, although it's not happening with GitHub. Seems to happen with Mozilla blogs. I'll open a few, leave Fenix, and come back to blank tabs. Could this be the same thing or a different issue?

**Comment by sblatz** [comment]
Similar to a comment raised in https://github.com/mozilla-mobile/fenix/issues/1002. This is currently not possible as we're waiting on a new GV API that will isolate each session.

@cpeterso any idea how far along that is?

**Comment by cpeterso** [comment]
GV needs to support e10s-multi so each tab has its own content process. bholley is investigating Android e10s-multi in Q3, but we don't have an ETA yet.

https://bugzilla.mozilla.org/show_bug.cgi?id=1530770

**Comment by sblatz** [comment]
I'm moving this to be a p5. We actually have a workaround for this right now which makes it so the behavior described above does not happen. We are restoring the other tabs in the background when a use presses either `close` or `restore` on the selected tab.

The only case that is a problem with this workaround is a **background tab** could crash, making it look like the currently selected one crashed.

**Comment by cpeterso** [comment]
For comparison, a Gecko crash in Fennec will crash the entire app. A Gecko crash in Fenix will crash all your tabs, but not the app.
